compiler/rustc_parse/src/parser/attr_wrapper.rs

```diff
@@ -439,12 +440,13 @@ fn make_token_stream(
                     continue;
                 }
 
+                let open_delim_sp = frame_data.open_delim_sp.unwrap();
```

Comment:
```suggestion
                let (open_delim, open_span) = frame_data.open_delim_sp.unwrap();
```
Instead of `.0`/`.1`.